enterprise/server/usage/usage_config/usage_config.go

```diff
@@ -0,0 +1,9 @@
+package usage_config
```

Comment:
can we call this config so the path names don't repeat?

(callers can choose to import as 'usage_config /path/to/usage/config' if they choose)